libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1109,11 +1109,11 @@ bool AP_Arming::rc_checks_copter_sub(const bool display_failure, const RC_Channe
         const RC_Channel *channel = channels[i];
         const char *channel_name = channel_names[i];
         // check if radio has been calibrated
-        if (channel->get_radio_min() > 1300) {
```

Comment:
There are two distinct uses of this pair of numbers (1300 and 1700).

One is setting strict endpoints for calibration values.

The other are a poor-man's `RC_OPTION` switch threshold values.

They should not be conflated.